src/components/Nav/UserInput/UserInput.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+// STYLES
+import * as S from './UserInputEle';
+
+function UserFormInputs({ form, getInputValue, inputValues }) {
+  console.log(`inputValues`, inputValues);
+  return (
+    <S.UserFromInputContainer>
+      <S.InputBox>
+        {form.inputLists.map((inputList, i) => {
+          return (
+            <>
+              <S.InputTextBox key={i}>
```

Comment:
```suggestion
            <Fragment key={i}>
              <S.InputTextBox>
```
프래그먼트가 최상위 태그일 때는 위와 같이 주셔야 합니다

Reply:
> 프래그먼트가 최상위 태그일 때는 위와 같이 주셔야 합니다

수정완료했습니다 !